python/test/t_SoizeGhanemFactory_std.expout

```diff
@@ -17,20 +17,65 @@ M=
-SoizeGhanem( 0 )= (1) * (([X0,X1]->1/sqrt(ComposedDistribution(Normal(mu = 0, sigma = 1), Uniform(a = -1, b = 1), NormalCopula(R = [[ 1   0.5 ]
- [ 0.5 1   ]])).computePDF(X0, X1))
+SoizeGhanem( 0 )= (1) * (([X0,X1]->1/sqrt(- ComposedDistribution
+- dimension=2
+- description= [X0,X1]
+- copula=NormalCopula(R = [[ 1   0.5 ]
+ [ 0.5 1   ]])
+
+| Index | Variable | Distribution |
+|-------|----------|--------------|
+|     0 |       X0 | Normal(mu = 0, sigma = 1) |
+|     1 |       X1 | Uniform(a = -1, b = 1) |
+).computePDF(X0, X1))
```

Comment:
after all it looks like the new ComposedDistribution::str doesnt mix well with the notation we had for distribution arithmetic
maybe it should be a new method to be called only when tabular formatting is requested ?

Reply:
Something similar to the _repr_html or _repr_markdown is in Python?

Reply:
- Would the name getMarkdown() be consistent?
- Providing a C++ implementation is required in the case the 'print' function is evaluated from Python. This does not trigger the html method, but calls the C++ str() method.

Reply:
rather __repr_markdown__ for consistency with the other ones, and also because it will stay hidden in the API

Reply:
Ok. Fixed at e5d294fbf.